README.md

```diff
@@ -1 +1,70 @@
-# E-Commerce App Using React
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
You have done docuemtian .But I kindly suggest you to update your content to real yours. I think you are using default `CRA` document

Reply:
Thankyou for your comment, noted and fixed